node/clients/Google.ts

```diff
@@ -4,20 +4,43 @@ import {
   PlaceDetailsRequest,
 } from '@googlemaps/google-maps-services-js'
 import { ExternalClient, IOContext, InstanceOptions } from '@vtex/api'
+import axios from 'axios'
 
 export class Google extends ExternalClient {
-  private externalClient: Client
+  private client: Client
+  private baseUrl = 'http://maps.googleapis.com'
+  private routes = {
+    placeAutocomplete: `${this.baseUrl}/maps/api/place/autocomplete/json`,
+    placeDetails: `${this.baseUrl}/maps/api/place/details/json`,
+  }
 
   constructor(context: IOContext, options?: InstanceOptions) {
     super('', context, options)
-    this.externalClient = new Client()
+
+    const axiosInstance = axios.create({
+      ...options,
+      baseURL: this.baseUrl,
+      headers: {
+        ...options?.headers,
+        'x-vtex-use-https': 'true',
+        'proxy-authorization': context.authToken,
+      },
+    })
+
+    this.client = new Client({ axiosInstance })
   }
 
   public placeAutocomplete(request: PlaceAutocompleteRequest) {
-    return this.externalClient.placeAutocomplete(request)
+    return this.client.placeAutocomplete({
+      ...request,
+      url: this.routes.placeAutocomplete,
```

Comment:
The lib seems to do nothing with the provided baseUrl :/, so the complete url must be passed on each request.